test/middlewares/validate.test.ts

```diff
@@ -34,7 +38,7 @@ const TEST_ASYNCAPI_DOCUMENT = new AsyncAPIDocument({
   }
 })
 
-const schema = TEST_ASYNCAPI_DOCUMENT.channel('test/channel').publish().message().payload().json()
+const schema = TEST_ASYNCAPI_DOCUMENT.channels().get('test/channel')?.json().publish.message.payload
```

Comment:
```suggestion
const schema = TEST_ASYNCAPI_DOCUMENT.channels().get('test/channel').operations().filterByReceive()[0].messages().all()[0].payload()
```

Reply:
I was running into an error in the validate function and now I realize is that in the docs it says that it returns `schema` [here](https://github.com/asyncapi/parser-api/blob/master/docs/api.md#message), but in the code it is returning `schemeInterface` [here](https://github.com/asyncapi/parser-js/blob/43a5b051b52ca87a519d66bb6c91307bf0f36556/src/models/v2/message.ts#L28) and tests are failing.